lib/funky/page.rb

```diff
@@ -0,0 +1,48 @@
+module Funky
+  class Page < GraphRootNode
+
+    # @example
```

Comment:
@bruce-fs `@example` in Yardoc normally identifies an example on how to use the code. A snippet of Ruby code that calls the method and displays the result.